modules/ollama/options.go

```diff
@@ -37,3 +37,28 @@ func withGpu() testcontainers.CustomizeRequestOption {
 		}
 	})
 }
+
+var _ testcontainers.ContainerCustomizer = (*UseLocal)(nil)
+
+// UseLocal will use the local Ollama instance instead of pulling the Docker image.
+type UseLocal struct {
+	env map[string]string
+}
+
+// WithUseLocal the module will use the local Ollama instance instead of pulling the Docker image.
+// Pass the environment variables you need to set for the Ollama binary to be used,
+// in the format of "KEY=VALUE". KeyValue pairs with the wrong format will cause an error.
+func WithUseLocal(keyVal map[string]string) UseLocal {
```

Comment:
suggestion: sorry to be a pain, I like your point about not having to pass nil here, so if we go for a key value pair here so it makes it slightly easier to use, constructing map from odd even items with a even number check that would be idea.

Reply:
I can align with that